crates/client/src/storage.rs

```diff
@@ -90,6 +99,28 @@ pub trait ContentStorage: Send + Sync {
     ) -> Result<DynHash>;
 }
 
+/// Represents information about a registry operator.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct OperatorInfo {
+    /// The last known checkpoint of the package.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub checkpoint: Option<SerdeEnvelope<MapCheckpoint>>,
```

Comment:
I don't think we need to store the checkpoint here. We store it per-package because not every package log is updated at once, whereas the operator log will always be updated with every request to update any package.

```suggestion
    /// The last known checkpoint of the package.
    #[serde(default, skip_serializing_if = "Option::is_none")]
    pub checkpoint: Option<SerdeEnvelope<MapCheckpoint>>,
```